aws-codeartifact-repository/overrides.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
since this is just overriding the values we pass the the contract tests, do we also need a version of this where instead of passing in a json object for the doc, we pass in a string version of this?

Reply:
we changed to only allow json objects, so we wouldnt need to test. Also I think this testing would be more suited for integration tests, these just needs to be  a dummy value so that we don't get a validation exception when running contract tests (I could've also set this to `null`). These tests mostly assert that handlers response with appropriate models.